src/artifact_providers/github.ts

```diff
@@ -110,25 +109,51 @@ export class GitHubArtifactProvider extends BaseArtifactProvider {
         lastArtifact.created_at >= revisionDate;
     }
 
-    if (checkNextPage) {
-      page += 1;
-    } else {
-      // If we are retrying, reset page index and start over
-      page = 0;
-      tries += 1;
-    }
```

Comment:
If you are removing this logic, then you should probably remove the `checkNextPage` logic above too, right? Or am I missing something?

Reply:
Ah, sorry I glanced over the for loop completely missing the use of `checkNextPage` there :)